src/site/layout/header.rs

```diff
@@ -3,7 +3,7 @@ use std::path::Path;
 use crate::config::artifacts::Artifacts;
 use crate::config::Config;
 use crate::errors::*;
-use crate::site::link;
+use crate::site::{link, Site};
```

Comment:
is there a reason you put the function in Site? i would probably put it in page...

Reply:
Honestly forgot about page.

Done some moving, let me know if it looks good